site/_posts/2016-11-28-fixing-bugs.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Last Week in ManageIQ: Fixing bugs and fixing bugs"
+author: skateman
+date: 2016-11-28 10:58:27 +01:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello! [Dávid](https://github.com/skateman) here from the UI team with
+some updates about the changes since last week.
+
+## Featured
+
+We are still focused on the stabilization of the upcoming Euwe release.
+In numbers this means 91 commits merged from 88 PRs created by 40 of our
+contributors since last week. If you are interested, check out the list
+of [open issues]((https://github.com/manageiq/manageiq/issues)) to help
+us in increasing these numbers.
+
+## Improved
+
+Chart widgets on the dashboard are sometimes displaying too many information
+on a too tiny place, making some data unreadable. There was a primitive
+zooming feature available, but it was not always helpful. Now thanks to
+our charts expert [Robin](https://github.com/PanSpagetka) the zoomed version
+uses [a full screen modal](https://github.com/ManageIQ/manageiq/pull/12811).
+
+There were also some improvements in the Automate part of our codebase.
+For example [Patrik](https://github.com/pkomanek) continued the [refactoring
+of automate methods](https://github.com/ManageIQ/manageiq/pull/12523). His
+victim was the `available_flavors` method for `miq_ae_orchestration`.
+
+## Fixed
+
+In these though days almost half of the number of PRs have the word "fix"
+in their title. This means it is extremely hard to select which one to
```

Comment:
possibly s/one/ones/ ?

Of course, there is no limit to how many fixes you can to cover here :wink: